django/contrib/admin/static/admin/css/base.css

```diff
@@ -276,24 +264,6 @@ tfoot td {
     border-top: 1px solid #eee;
 }
 
-thead th:first-child,
-tfoot td:first-child {
-    border-left: none !important;
-}
-
-thead th.optional {
-    font-weight: normal !important;
-}
-
-tr.row-label td {
-    font-size: 9px;
-    padding-top: 2px;
-    padding-bottom: 0;
-    border-bottom: none;
-    color: #666;
-    margin-top: -1px;
-}
-
```

Comment:
Didn't find any mention of the `row-label` and `optional` in the project. Removed.